source/install/cloud-changelog.md

```diff
@@ -4,12 +4,61 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-03-16](#release-2022-03-16)
 - [Release 2022-03-08](#release-2022-03-08)
 - [Release 2022-03-02](#release-2022-03-02)
 - [Release 2022-02-16](#release-2022-02-16)
 - [Release 2022-02-10](#release-2022-02-10)
 - [Release 2022-02-03](#release-2022-02-03)
-- [Release 2022-01-27](#release-2022-01-27)
+
+## Release 2022-03-16
+
+### Compatibility
+ - Updated Safari recommended minimum version to v14.1+.
+
+### Improvements
+
+#### User Interface (UI)
+ - The support email field was from "Customization" to "Notifications" in the System Console. Also, the support email is now required to be set when configuring email notifications.
+ - The ping endpoint can now receive a device ID, which will report whether the device is able to receive push notifications.
+ - Added a loading indicator to the Threads global list each time more posts are fetched on infinite scroll.
+ - Added a search guidance to the Threads global list when no more posts can be loaded. This is only shown if you’ve scrolled to load older posts and then reach the end of the list.
+ - Added support for inline editing of posts.
```

Comment:
Reminder that this requires a feature flag to be enabled in production in Split.io

Reply:
@amyblais are you taking care of that? I'll open the tix to remove the feature flag for the april on-prem release

Reply:
@esethna I've let Michel and Jelena know that the feature flag can be enabled for Cloud test servers https://community.mattermost.com/core/pl/hrpjugbec3dtzx9f4zx57nohrr.